src/ownership-and-deconstruction.rst

```diff
@@ -258,65 +258,47 @@ Passing Conventions
-from a :t:`function`.
-
-:dp:`fls_1gyeqfpe7m1m`
-A :t:`value` is subject to a :t:`passing convention` when the :t:`value` is
-
-* :dp:`fls_jag0ud2lv08`
-  Assigned using an :t:`assignment expression`,
-
-* :dp:`fls_bkuz12srez4`
-  Bound to a :t:`function parameter` in a :t:`call expression`,
-
-* :dp:`fls_owltkb5i2lah`
-  Bound to a :t:`pattern`,
-
-* :dp:`fls_olqpm32j8va6`
-  Captured by a :t:`capture expression`,
-
-* :dp:`fls_czune894326w`
-  Returned from a :t:`function`.
-
-* :dp:`fls_qi148dixkp0w`
-  **More?**
+A :t:`passing convention` is the mechanism that defines how a :t:`value` is
+transferred between :t:`[place]s`. **place is not the appropriate term here**
 
 :dp:`fls_h2pgsij1rbms`
-A :t:`by copy type` is a :t:`type` that implements the :std:`core::marker::Copy`
+A :t:`copy type` is a :t:`type` that implements the :std:`core::marker::Copy`
 :t:`trait`.
 
 :dp:`fls_yx2knbby70fy`
-A :t:`value` of a :t:`by copy type` is passed :dt:`by copy`. Passing :t:`by
-copy` does not change the :t:`owner` of the :t:`value`.
+A :t:`value` of a :t:`copy type` is :t:`passed <passing convention>` :dt:`by
+copy`. Passing :t:`by copy` does not change the :t:`owner` of the :t:`value`.
 
 :dp:`fls_6ul3f6v0foma`
-A :t:`by move type` is a :t:`type` that does not implement the
-:std:`core::marker::Copy` :t:`trait`.
+A :t:`move type` is a :t:`type` that implements the :std:`core::marker::Sized`
+:t:`trait` and that is not a :t:`copy type`.
 
 :dp:`fls_3ztdz02efeoc`
-A :t:`value` of a :t:`by move type` is passed :dt:`by move`. Passing :t:`by
-move` changes the :t:`owner` of the :t:`value`.
+A :t:`value` of a :t:`move type` is :t:`passed <passing convention>` :dt:`by
+move`. Passing :t:`by move` changes the :t:`owner` of the :t:`value`.
+
+:dp:`fls_konzgoybhfqm`
+A :t:`value` of a :t:`place expression` shall be :t:`passed <passing
+convention>` :t:`by move` only when it denotes:
 
-:dp:`fls_ljc1k8ms16gp`
-Passing :dt:`by value` is either passing :t:`by copy` or passing :t:`by move`.
+* :dp:`fls_4bnbv7mqod57`
+  A :t:`field` of a :t:`place expression` that can be :t:`passed <passing
+  convention>` :t:`by move` and whose type does not implement the
+  :std:`core::ops::Drop` :t:`trait`, or
 
-:dp:`fls_xis57dxiomwz`
-A :t:`value` of a :t:`reference type` is passed :dt:`by reference`. Passing
-:t:`by reference` temporarily changes the :t:`owner` of the :t:`value`.
+* :dp:`fls_3xk3p1unbjy5`
+  A :t:`temporary`, or
 
-:dp:`fls_JUNnLwhPSwUO`
-Passing :dt:`by immutable reference` is passing :t:`by reference` where the
-:t:`value` is :t:`immutable`.
+* :dp:`fls_vk1xhvdaakh0`
+  A :t:`variable` which is not currently :t:`borrowed`.
 
-:dp:`fls_WW0GvbiEnyiW`
-Passing :dt:`by unique immutable reference` is passing
-:t:`by immutable reference` where it is asserted that the :t:`reference` to
-the :t:`value` is the only live :t:`reference`.
+:dp:`fls_vveEJn7lngT8`
+A :t:`value` of a :t:`value expression` is always :t:`passed <passing
```

Comment:
I prefer not to split references to terms across two lines as this makes searching hard.